src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -4541,16 +4552,21 @@ internal string GetAbsolutePath(string moduleBase, string path)
             }
         }
 
+        /// <summary>
+        /// Check if a path is rooted or "relative rooted".
+        /// </summary>
+        /// <param name="filePath">The file path to check.</param>
+        /// <returns>True if the path is rooted, false otherwise.</returns>
         internal static bool IsRooted(string filePath)
         {
-            return (Path.IsPathRooted(filePath) ||
+            return Path.IsPathRooted(filePath) ||
                 filePath.StartsWith(@".\", StringComparison.Ordinal) ||
```

Comment:
Also the .NET team now recommends `Path.IsFullyQualified()` rather than `Path.IsRooted()`.

Reply:
Why not replace `Path.IsPathRooted` with `Path.IsPathFullyQualified`?

Reply:
I don't depend on this method and don't want to break anything. Just wanted to note it here. Either way, it didn't help my needs since this method counts `"../here"` as "rooted".

An earlier version of my fix changed this method and used it, but I decided it's better not to touch too much.

Reply:
> it didn't help my needs since this method counts "../here" as "rooted".

This is what I see:
```powershell
PS:2> [System.IO.Path]::IsPathRooted("../here")
False
PS:3> [System.IO.Path]::IsPathFullyQualified("../here")
False
```
I'm mainly curious about the different between `Path.IsPathRooted` and `Path.IsPathFullyQualified`. If there is no difference in behavior, then we should follow the guidance and replace it in our code.

Reply:
Sorry I meant the `ModuleCmdletBase.IsRooted("../here")` method.

I didn't change the body in terms of how it works just in case I break it.

But agreed, we should probably change it.

There is a similar new change in .NET Core 2.1 with `Path.Join()` vs `Path.Combine()` which seems to just be a `ReadOnlySpan<char>` API difference, but we should look into it.

Reply:
I've put the parens back into `ModuleCmdletBase.IsRooted` so that my PR doesn't touch it (since I don't use it now).

I think we should move over to `Path.IsFullyQualified` but do it in another PR. I haven't looked at `ModuleCmdletBase.IsRooted` very much here and haven't added any testing for regressions in it.